api/textString.js

```diff
@@ -8,27 +8,35 @@ function getManyTextStrings(condition = {}) {
   return TextString.find(condition);
 }
 
-function createTextString(hebrew) {
-  return TextString.create({
-    hebrew,
-    english: "",
-  });
+function createTextString(body, language) {
+  let string =
+    language === "hebrew"
```

Comment:
I don't think you need this check, you can just do:
`string = {[language]: body}`

I do think we need to validate it here, but do it separately - have a function called `validateLanguage` and just check that the `language` parameter exists in the supported languages